wpilib/wpilib/driverstation.py

```diff
@@ -231,19 +323,22 @@ def getStickAxis(self, stick: int, axis: int) -> float:
 
         :returns: The value of the axis on the joystick.
         """
-        if axis < 0 or axis >= hal.kMaxJoystickAxes:
+        if 0 > axis or axis >= hal.kMaxJoystickAxes:
             raise IndexError("Joystick axis is out of range")
         if stick < 0 or stick >= self.kJoystickPorts:
             raise IndexError("Joystick index is out of range, should be 0-%s" % self.kJoystickPorts)
 
         with self.cacheDataMutex:
             joystickAxes = self.joystickAxes[stick]
 
-            if axis >= joystickAxes.count:
-                self._reportJoystickUnpluggedWarning("Joystick axis %d on port %d not available, check if controller is plugged in\n" % (axis, stick))
-                return 0.0
+            axis_unavailable = axis >= joystickAxes.count
+
+            if not axis_unavailable:
```

Comment:
Seems redundant. This line is nowhere close to being long.

Reply:
Yeah, it's not necessary here.